packages/e2e-test-utils/src/click-placeholder-button.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Clicks a button in a placeholder based on the label text.
+ *
+ * @param {string} buttonText The text that appears on the button to click.
+ */
+export async function clickPlaceholderButton( buttonText ) {
+	const _button = await page.waitForFunction(
+		( text ) => {
+			const placeholders = document.querySelectorAll(
+				'.wp-block-editor-placeholder'
+			);
+
+			for ( const placeholder of placeholders ) {
+				const buttons = placeholder.shadowRoot.querySelectorAll(
+					'button,label,[aria-label]'
+				);
+
+				for ( const button of buttons ) {
+					if (
+						button.textContent === text ||
+						button.getAttribute( 'aria-label' ) === text
+					) {
+						return button;
```

Comment:
Should we make sure only one element matches `buttonText`?

Reply:
This is the same pattern we've been using elsewhere (`clickButton` and `clickBlockToolbarButton`). It clicks the first match. I think it's fine?

Reply:
Yeah, I guess you're right.